### ISSUE TITLE
Wrong number of arguments (3 for 2)

### ISSUE THREAD

**Issue by mareczek** [globalize/globalize-accessors#29]
Whenever I send params, such as:
```ruby
{
  name_en: "English name"
  name_de: "German name"
}
```
and in the controller:
```ruby
@model.update_attributes(params)
```
i get:
`ArgumentError (wrong number of arguments (3 for 2)):`

globalize-accessors is at version 0.2.1

**Comment by mareczek** [comment]
The problem occurs when used with gem `enumerize`

**Comment by abuisman** [comment]
@mareczek I have just gotten the same error. I have not yet investigated what I need to do to fix it though. Have you found a solution/cause?

**Comment by mareczek** [comment]
I did not solve the problem. Both enumerize and globalize overwrite the same method, hence the problem.

My temp solution working until now was to not use both gems together in the same model.

**Comment by abuisman** [comment]
Morgoth came to me in a vision and helped me produce this workaround:

````ruby
  def write_attribute(name, value, options = {})
    return Globalize::ActiveRecord::InstanceMethods.instance_method(:write_attribute).bind(self).call(name, value, options) if translated?(name)
    super
  end
````

You can put this in the class that has both enumerize and globalize.

**Comment by shioyama** [comment]
Hmm.. I don't seem to be able to reproduce this. I can update attributes fine with both globalize (latest on `master` branch) and enumerize (2.0.0). Can either of you provide a concrete example?

**Comment by shioyama** [comment]
This has been fixed in brainspec/enumerize#236, closing.
